backend-project/small_eod/generic/tests.py

```diff
@@ -74,6 +74,28 @@ def get_create_data(self):
         return data
 
 
+class ReadOnlyPaginatedViewSetMixin(ReadOnlyViewSetMixin):
+    def test_list_plain(self):
```

Comment:
Co to? Niby jest test, a nic nie robi? Czyli tylko wprowadzenie w błąd dewelopera, który spojrzy, że działa ```test_list_plain```? Proponuje zmodyfikować dotychczasową implementacje ```test_list_plain```, aby uwzględniała ```paginated=True```, zamiast takich rzeczy.